portfolio/components/Navbar.jsx

```diff
@@ -0,0 +1,104 @@
+'use client'
+import Link from 'next/link'
+import Image from 'next/image';
+import { useState } from 'react'
+import { logo, sunIcon, moonIcon, hamburger, cross } from '../assets';
+import { useTheme } from 'next-themes';
+
+const MenuItems = ({ isMobile, active, setActive, setIsOpen }) => {
+    const navLinks = [
+        { url: '/', text: 'Home' },
+        { url: 'case-studies', text: 'Case Studies' },
+        { url: 'contact', text: 'Contact' },
+        { url: 'resume', text: 'Resume' },
+    ]
+
+    return (
+        <div className="flex flex-row justify-between list-none gap-4 items-center">
+            <ul className={`list-none flex ${isMobile
+                ? 'flex-col justify-end w-full border-b border-black'
+                : 'flex flex-row justify-between list-none gap-4'
+                }`}>
+                {navLinks.map((link, i) => (
```

Comment:
One Note: 
Using the index as a key in React is generally not recommended, as it can impact performance negatively and could lead to some unstable component behavior when items are added, removed, or reordered. It is important to choose unique keys that are unlikely to change, as React will not be able to track changes to the state if the keys change.

In your case, I don't see any issues that would arise but keep this in mind in the future.